docs/reference/common-xref.md

```diff
@@ -0,0 +1,31 @@
+# Common dbxref and their shorthands
+
+This is a list of commonly used xrefs shorthands. For full list, please refer to https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.yaml
```

Comment:
I'd replace this with the following text on database_cross_references:

"database_cross_references" are used as a flexible way of making mappings without any semantics.  There are two major types of mappings

(a) Mapping to database identifiers in the form of strings with syntax {prefix}:{accession}. Some of these string also correspond to CURIES. The current reference for these prefixes is https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.yaml.
(b) Mapping to other ontologies.  These take the form of a standard OBO compact URI, e.g. http://purl.obolibrary.org/obo/CL_0000100 --> CL:0000100, stored as a string*. This form of mapping is slowly being superseded by the much richer [SSSOM format]().

* The standard typing of this field has been controversial and may evolve in future.

Reply:
I've removed this whole document. I think you are right that for now we just redirect to the yaml file. I think it is sufficient for now to leave it in the table as something like what you suggested: 
```
string and should* take the form {prefix}:{accession}; see [db-xrefs yaml](https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.yaml) for prefixes
```